README.md

```diff
@@ -52,7 +52,7 @@ By default **semantic-release** uses [Angular Commit Message Conventions](https:
 
 Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint](https://github.com/conventional-changelog/commitlint) can be used to help contributors and enforce valid commit messages.
 
-Here is an example of the release type that will be done based on a commit messages:
+The table below shows which commit message gets you which release type when `semantic-release` runs:
```

Comment:
maybe we should add "using the default configuration", as commits can be configured using [`@semantic-release/commit-analyzer`](https://github.com/semantic-release/commit-analyzer) options?

Reply:
```suggestion
The table below shows which commit message gets you which release type when `semantic-release` runs (using the default configuration):
```